src/mod2c_core/deriv.c

```diff
@@ -187,12 +187,6 @@ method->name, fun->name, suffix, method->name, fun->name,
 suffix, ssprefix, method->name,
 numeqn, listnum, listnum, method->name, fun->name, suffix);
 	vectorize_substitute(qsol, buf);
-	Sprintf(buf,
-	  "\n"
-	  "#pragma acc routine seq\n"
-	  "extern int %s%s_thread(int, int*, int*, int, _threadargsproto_);\n"
-	  , ssprefix, method->name);
```

Comment:
make sure extern declarations in coreneuron has `acc routine seq`
(haven't reviewed coreneuron yet).

Reply:
I will double chekc that